src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -159,8 +167,13 @@ class Docker implements Serializable {
         public void tag(String tagName = parsedId.tag, boolean force = true) {
             docker.node {
                 def taggedImageName = toQualifiedImageName(parsedId.userAndRepo + ':' + tagName)
-                // TODO as of 1.10.0 --force is deprecated; for 1.12+ do not try it even once
-                docker.script.sh "docker tag --force=${force} ${id} ${taggedImageName} || docker tag ${id} ${taggedImageName}"
+                // as of 1.10.0 --force is deprecated; for 1.12+ do not try it even once
```

Comment:
Comment is now misleading. For 1.10+ we do not try it.

Reply:
It was misleading before ... but I'm happy to change this.

Reply:
I think the source of the complaints (and now 2PRs https://github.com/jenkinsci/docker-workflow-plugin/pull/122) for this is that it shows up as an error in the log, so when things go wrong, people see it as a problem (even though it wasn't)